include/deal.II/particles/particle.h

```diff
@@ -25,63 +25,65 @@
 DEAL_II_NAMESPACE_OPEN
 
 /**
- * A namespace that contains all classes that are related to the particle
- * implementation, in particular the fundamental Particle class.
+ * A namespace for all type definitions related to particles.
```

Comment:
Can you please verify that this namespace description does not clash with [this one](http://dealii.org/developer/doxygen/deal.II/namespacetypes.html)? If so then I think that this description can simply be removed.